pkg/security/securitycontextconstraints/selinux/mustrunas_test.go

```diff
@@ -79,43 +79,76 @@ func TestMustRunAsValidate(t *testing.T) {
 	seType := newValidOpts()
 	seType.Type = "invalid"
 
+	levelWithIdenticalSensitivity := newValidOpts()
+	levelWithIdenticalSensitivity.Level = "s0-s0:c6,c0"
+
+	levelWithDifferentOrderOfCategories := newValidOpts()
+	levelWithDifferentOrderOfCategories.Level = "s0:c0,c6"
+
+	levelWithAbbreviatedCategories := newValidOpts()
+	levelWithAbbreviatedCategories.Level = "s0:c0.c3"
+
+	levelWithContiguousSeriesOfCategories := newValidOpts()
+	levelWithContiguousSeriesOfCategories.Level = "s0:c0,c1,c2,c3"
+
 	tests := map[string]struct {
-		seLinux     *api.SELinuxOptions
-		expectedMsg string
+		seLinux         *api.SELinuxOptions
+		expectedSeLinux *api.SELinuxOptions
+		expectedMsg     string
 	}{
 		"invalid role": {
-			seLinux:     role,
-			expectedMsg: "does not match required role",
+			seLinux:         role,
+			expectedSeLinux: newValidOpts(),
+			expectedMsg:     "does not match required role",
 		},
 		"invalid user": {
-			seLinux:     user,
-			expectedMsg: "does not match required user",
+			seLinux:         user,
+			expectedSeLinux: newValidOpts(),
+			expectedMsg:     "does not match required user",
 		},
 		"invalid level": {
-			seLinux:     level,
-			expectedMsg: "does not match required level",
+			seLinux:         level,
+			expectedSeLinux: newValidOpts(),
+			expectedMsg:     "does not match required level",
 		},
 		"invalid type": {
-			seLinux:     seType,
-			expectedMsg: "does not match required type",
+			seLinux:         seType,
+			expectedSeLinux: newValidOpts(),
+			expectedMsg:     "does not match required type",
 		},
 		"valid": {
-			seLinux:     newValidOpts(),
-			expectedMsg: "",
+			seLinux:         newValidOpts(),
+			expectedSeLinux: newValidOpts(),
+			expectedMsg:     "",
+		},
+		"valid level with identical sensitivity": { // "s0:c6,c0" == "s0-s0:c6,c0"
+			seLinux:         levelWithIdenticalSensitivity,
+			expectedSeLinux: newValidOpts(),
+			expectedMsg:     "",
+		},
+		"valid level with different order of categories": { // "s0:c6,c0" == "s0:c0,c6"
+			seLinux:         levelWithDifferentOrderOfCategories,
+			expectedSeLinux: newValidOpts(),
+			expectedMsg:     "",
+		},
+		"valid level with abbreviated categories": { // "s0:c0.c3" == "s0:c0,c1,c2,c3"
```

Comment:
need some additional tests, test edges cases and multiple equivalences.

Reply:
What do you mean by "multiple equivalences"?